public/app/features/scenes/builders/panelBuilders.ts

```diff
@@ -1,8 +1,8 @@
 import { VizPanel, VizPanelState } from '@grafana/scenes';
 import { GraphFieldConfig, TableFieldOptions } from '@grafana/schema';
-import { PanelOptions as BarGaugePanelOptions } from 'app/plugins/panel/bargauge/panelcfg.gen';
-import { PanelOptions as TablePanelOptions } from 'app/plugins/panel/table/panelcfg.gen';
-import { PanelOptions as TimeSeriesOptions } from 'app/plugins/panel/timeseries/panelcfg.gen';
+import { Options as BarGaugePanelOptions } from 'app/plugins/panel/bargauge/panelcfg.gen';
+import { Options as TablePanelOptions } from 'app/plugins/panel/table/panelcfg.gen';
+import { Options as TimeSeriesOptions } from 'app/plugins/panel/timeseries/panelcfg.gen';
```

Comment:
I know you haven't changed these, but the inconsistency in FooPanelOptions vs BarOptions irks me

Reply:
agreed - i think this is something where we can increase consistency if we generate a TS module containing the types, rather than just bare-exporting them within the root plugin directory like we do today. Somewhat analogous to what we do in `packages/grafana-schema`, in particular with [the generated index file](https://github.com/grafana/grafana/blob/main/packages/grafana-schema/src/index.gen.ts)

Reply:
#64723 is relevant to that effort